redash/handlers/data_sources.py

```diff
@@ -75,8 +75,10 @@ def post(self):
 
 class DataSourceSchemaAPI(BaseResource):
     def get(self, data_source_id):
+        refresh = request.args.get("refresh", "0", type=bool)
```

Comment:
Fixed it. 

In Python 2.x, True is 1 and 0 is False, however it can get overridden (unlike Python 3.x).
http://stackoverflow.com/questions/2764017/is-false-0-and-true-1-in-python-an-implementation-detail-or-is-it-guarante

But its better to be safe than sorry.